app/models/rbac.rb

```diff
@@ -302,17 +302,17 @@ def self.minimum_columns_for(klass)
     columns.join(', ')
   end
 
-  def self.get_user_info(userid, miq_group_id = nil)
-    user                      = User.find_by_userid(userid)
-    miq_group                 = MiqGroup.find_by_id(miq_group_id)
+  def self.get_user_info(user, userid, miq_group, miq_group_id)
```

Comment:
Since we are changing the callers anyway, why keep the user_id and group_id parameters?